target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -36,10 +37,21 @@
 
 	soc {
 		pinmux@800000 {
-			i2c4_pins: i2c4_pinmux {
-				pins = "gpio12", "gpio13";
-				function = "gsbi4";
-				bias-disable;
+			led_pins: led_pins {
+				
+				pins = "gpio6", "gpio7", "gpio8", "gpio9", "gpio26", "gpio33",
+					"gpio53", "gpio66", "gpio59", "gpio60", "gpio61",
```

Comment:
Gpios 59, 60, 61 and 62 a RGMII gpios (check relative node in DTS)

Also 26 and 33 are output-high
6, 7, 8, 9, 53 and 66 are output-low